.github/workflows/tests.yml

```diff
@@ -0,0 +1,65 @@
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - 'stable/**'
+
+name: Tests
+
+jobs:
+  windows_and_macos:
+    runs-on: ${{ matrix.os }}
```

Comment:
This is a common way to do things, but I'm not a fan. I don't believe linting errors should block builds. 

It's a poor experience: you push, you've got some silly whitespace issue, you fix and push it again, then your tests fail. Rinse and repeat.

It seems better to me to get a full set of failures (tests and linting errors) in a single step rather than piecemeal.